### PR TITLE
PBI-7282 Use 6.9 and bump version and gradle compatibility

### PR DESCRIPTION
**Pull request by PoornimaApptentive**

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -46,8 +46,8 @@ class _MyAppState extends State<MyApp> {
     final String apptentiveKey;
     final String apptentiveSignature;
     if (Platform.isAndroid) {
-      apptentiveKey = "<YOUR_ANDROID_KEY>";
```
Comment (frankus) [inline example/lib/main.dart:49]: Want to remove this!

**Review by frankus — CHANGES_REQUESTED** [review]
Remove API credentials. Looks good otherwise.

File: example/lib/main.dart (hunk 1 above)
Comment (frankus) [reply example/lib/main.dart:49]: Sorry, should have commented on the right side of the diff, but there are FoxFax credentials in the diff.

Comment (PoornimaApptentive) [reply example/lib/main.dart]: Thanks for catching that! I have updated the PR.

**Review by frankus — APPROVED** [review]

**Pull request merged**